doc/audit/read_write_restrictions.rst

```diff
@@ -0,0 +1,78 @@
+Read-Write Permissions On KV Maps
+=================================
+
+To allow governance audit we ensure that all governance actions can be viewed and understood from the ledger, and signed by members. To support this CCF distinguishes governance tables from application tables, and restricts which tables can be read from and written to in different execution contexts.
+
+Table Namespaces
+----------------
+
+There are 3 categories of table defined:
+
+- Governance tables. These begin with a ``public:ccf.gov`` prefix, indicating that they are public (their contents are not encrypted in the ledger), and in the ``ccf.gov`` namespace. These should only be modified by constitution-controlled governance actions.
+- Internal tables. These begin with a ``ccf.internal`` or ``public:ccf.internal`` prefix, indicating that they are in the ``ccf.internal`` namespace, and may be public or private. These can be considered a lower-level subset of governance tables, though some are also modified by node-triggered operations such as signatures, rather than governance actions.
```

Comment:
We can clarify that they are managed by CCF itself, and that direct user writes are completely unsupported.